updating/understanding-openshift-updates.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Understanding {product-title} updates
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (`oc`). Platform administrators are automatically notified when an update is available for their cluster. Updates can be applied either from the `oc` command-line interface (CLI) or the web console.
```

Comment:
I think we can drop the final sentence here, now that the first sentence covers the UI choices.